src/CoreBundle/Resources/public/vendor/moment/moment.d.ts

```diff
@@ -196,11 +196,11 @@ declare namespace moment {
     toJSON(): string;
 
     /**
-     * @deprecated since version 2.8.0
+     * @deprecated since sonata-project/core-bundle 2.8.0
```

Comment:
These annotations are not from "sonata-project/core-bundle", but from the 3rd party NPM library.

Reply:
Good catch